tests/compute/test_basics.py

```diff
@@ -290,6 +296,68 @@ def _check_nx_feature(nxg, nf, ef):
     assert F.allclose(g.edata['h'], F.tensor([[1., 2.], [1., 2.],
                                               [2., 3.], [2., 3.]]))
 
+def test_to_cugraph_conversion():
+    if cugraph is None:
+        pytest.skip("Skipping to_cugraph test because cugraph is not installed")
```

Comment:
```suggestion
    cugraph = pytest.importorskip("cugraph", reason="Skipping to_cugraph test because cugraph is not installed")
```
If you do this for tests that use `cugraph`, then you can also get rid of the `try: except:` import of `cugraph` above.

Reply:
Agree, Resolved with https://github.com/rapidsai/dgl/pull/13/commits/df9152d2978017ad333efd06542acd894c4201dc